PurchasesCoreSwift/IntroEligibilityCalculator.swift

```diff
@@ -94,6 +103,7 @@ enum IntroEligibilityStatus: Int {
 
 extension IntroEligibilityStatus {
     func toNSNumber() -> NSNumber {
+        // swiftlint:disable:next compiler_protocol_init
         return NSNumber(integerLiteral: self.rawValue)
```

Comment:
You can switch this implementation to
`return self.rawValue as NSNumber`

Reply:
Great I wasn't sure so I just ignored it hahaha